templates/search/blocks/visualisation_menu.html

```diff
@@ -23,9 +23,9 @@
                         <span class="search-visualisation-nav__link-label">Timeline view</span>
                     </a>
                 </li>
-                <li class="search-visualisation-nav__list-item">
+                <li class="search-visualisation-nav__list-item custom_icon">
                     <a href="{{tag_view_url}}" class="search-visualisation-nav__link"{% if form.vis_view.value == vis_view.TAG.value %} aria-current="true"{% endif %}>
-                        <i class="fa-solid fa-circle fa-xl"></i>
+<!--                        <i class="fa-solid fa-circle fa-xl"></i>-->
```

Comment:
same as above on commented. use {% comment %} template tags for Non JS. 
Also see comment on `tag.svg`.

Reply:
I've removed the commented code.